cmd/run.go

```diff
@@ -107,6 +107,8 @@ func (o *runOption) preRunE(cmd *cobra.Command, args []string) (err error) {
 		o.reportWriter = runner.NewHTMLResultWriter(writer)
 	case "json":
 		o.reportWriter = runner.NewJSONResultWriter(writer)
+	case "pdf":
+		o.reportWriter = runner.NewPDFResultWriter(writer)
```

Comment:
You could ask the users input the PDF filename with `o.reportFile`. So, the PDF writer could write the data into a user-defined file. For instance:

`o.reportWriter = runner.NewPDFResultWriter(o.reportFile, writer)`